tests/test_plugins_os_unix_log.py

```diff
@@ -97,3 +100,28 @@ def test_atop_plugin(target_unix, fs_unix):
     assert bool(results[0].wasinactive) is False
     assert results[0].container == ""
     assert str(results[0].filepath) == "atop_20221111"
+
+
+def test_audit_plugin(target_unix, fs_unix):
```

Comment:
Can you also move this test into a separate `test_plugins_os_unix_log_audit.py`?

I actually want the other tests in here also into separate files but we'll fix that later.